package.json

```diff
@@ -92,7 +92,7 @@
     "clean-all": "yarn clean-e2e && yarn build-clean && rimraf --glob './packages/*/node_modules' && rimraf './node_modules'",
     "clean-e2e": "node ./scripts/cleanE2e.mjs",
     "crowdin:upload": "echo 'Uploading sources to Crowdin' && crowdin upload sources --config ./crowdin.yaml",
-    "crowdin:download": "echo 'Downloading translations from Crowdin' && crowdin download --config ./crowdin.yaml",
+    "crowdin:download": "echo 'Downloading translations from Crowdin' && crowdin download --config ./crowdin.yaml --language ja --language es-ES --language fr --language pt-BR --language ro --language ru --language uk --language zh-CN",
```

Comment:
This can help speed up your Netlify prod deployment a bit.
I doubt you plan to introduce any new language so this should be fine 🤷‍♂️